com.unity.netcode.gameobjects/Runtime/Core/SnapshotSystem.cs

```diff
@@ -210,6 +267,19 @@ internal void WriteSpawn(ClientData clientData, NetworkWriter writer, in Snapsho
             writer.WriteUInt16(spawn.TickWritten);
         }
 
+        internal void WriteDespawn(ClientData clientData, NetworkWriter writer, in SnapshotDespawnCommand despawn)
+        {
+            // remember which spawn we sent this connection with which sequence number
+            // that way, upon ack, we can track what is being ack'ed
+            ClientData.SentSpawn s;
+            s.ObjectId = despawn.NetworkObjectId;
+            s.Tick = despawn.TickWritten;
+            s.SequenceNumber = clientData.SequenceNumber;
+            clientData.SentSpawns.Add(s);
+
+            writer.WriteUInt64(despawn.NetworkObjectId);
+            writer.WriteUInt16(despawn.TickWritten);
```

Comment:
Ticks are now `ulongs` in the new tick system. I think we might need to adjust this here?